// Reversed by pbd (https://github.com/rsc-dev/pbd)
package Devialet.IMASlave4U;
import "CallMeMaybe/CallMeMaybe.proto";

service SoundControl {
	option (Devialet.CallMeMaybe.dvltServiceOptions) = {
		serviceName: "com.devialet.imaslave4u.soundcontrol-0"
		properties {
		  property {
		    type: "Devialet.CallMeMaybe.BoolProperty"
		    name: "nightMode"
		  }
		  property {
		    type: "Devialet.CallMeMaybe.DoubleProperty"
		    name: "volume"
		    logUpdate: false
		  }
		}
		
	};
}